build/dockerfiles/Dockerfile

```diff
@@ -25,5 +25,6 @@ COPY README.md .htaccess /usr/local/apache2/htdocs/
 COPY output/v3 /usr/local/apache2/htdocs/v3
 COPY v3/plugins/.htaccess /usr/local/apache2/htdocs/v3/plugins/
 COPY v3/images/eclipse-che-logo.png /usr/local/apache2/htdocs/v3/images/
+COPY v4 /usr/local/apache2/htdocs/v4/
```

Comment:
we don't need v4 if one are meta.yaml and the other one are devfile.yaml

Reply:
I think we need it.
There is no technical need, but it seems reasonable to me, to make it clear that the API is changed.